test/epiviewpoint_web/controllers/import_controller_test.exs

```diff
@@ -87,14 +87,68 @@ defmodule EpiViewpointWeb.ImportControllerTest do
       assert conn |> redirected_to() == "/import/start"
       assert "Invalid mm-dd-yyyy format: 06/02/bb" = Session.get_import_error_message(conn)
     end
+
+    test "accepts ndjson file upload", %{conn: conn} do
+      temp_file_path =
+        """
+        {"search_firstname_2":"Alice","search_lastname_1":"Testuser","dateofbirth_8":"01/01/1970","datecollected_36":"06/02/2020","resultdate_42":"06/01/2020","datereportedtolhd_44":"06/03/2020","result_39":"positive","glorp":"393","person_tid":"alice"}
+        {"search_firstname_2":"Billy","search_lastname_1":"Testuser","dateofbirth_8":"03/01/1990","datecollected_36":"06/05/2020","resultdate_42":"06/06/2020","datereportedtolhd_44":"06/07/2020","result_39":"negative","glorp":"sn3","person_tid":"billy"}
+        """
+        |> Tempfile.write_ndjson!()
```

Comment:
Consider using [ExUnit's tmp dir](https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-tmp-dir) and then you don't need the `File.rm/1`.

Reply:
updated.